rom-elks.c

```diff
@@ -381,11 +382,11 @@ static int int_13h ()
 
 		default:
 			printf ("fatal: INT 13h: AH=%hxh not implemented\n", ah);
-			assert (0);
+			return err;
 		}
 
 	flag_set (FLAG_CF, err? 1: 0);
-	return err;
+	return 0;
```

Comment:
Interrupt hooks like this one **must** return an error to EMU86: 0 for OK, < 0 to stop the emulator, or > 0 to tell the emulator to try the real interrupt. This latest allows to implement only the functions that one wants to tweak if testing with a real BIOS, like a kind of 'BIOS overlay'.

Reply:
I understand completely. That's what the "return err" is for on line 385, which returns -1 on unimplemented subfunctions. This code does what you are requesting with EMU86.

Regarding incorrect disk parameters, ELKS won't boot without being able to query non-existent disks. It won't boot without this change.

The "default" case in the switch handles the request to find BIOS bugs.

Reply:
As you want, that's an `*-elks.c` file, so intented for ELKS maintainer.